example/ck_tile/01_fmha/fmha_fwd.cpp

```diff
@@ -177,10 +179,16 @@ bool run(const ck_tile::ArgParser& arg_parser)
         return false;
     }
 
-    ck_tile::index_t seqlen_q = arg_parser.get_int("s");
-    ck_tile::index_t seqlen_k = arg_parser.get_int("s_k");
-    if(seqlen_k < 0)
-        seqlen_k = seqlen_q;
+    auto [seqlen_qs, seqlen_ks] =
+        decode_seqlen(mode, batch, arg_parser.get_str("s"), arg_parser.get_str("s_k"));
+
+#if 0
```

Comment:
do we need a macro to turn on/off debug prints?

Reply:
that is for debug purpose :)